src/importexport/mei/tests/mei_tests.cpp

```diff
@@ -68,8 +76,18 @@ void Mei_Tests::meiReadTest(const char* file, const char* ext)
         return;
     }
 
-    EXPECT_TRUE(ScoreComp::saveCompareScore(score, fileName + u".mscx", MEI_DIR + fileName + u".mscx"));
+    // Save the .mei file to the ./tmp directory for round trip testing
+    bool output = ScoreRW::saveScore(score, ScoreRW::rootPath() + u"/" + TMP_DIR + fileName + u".mei", exportFunc);
```

Comment:
I'm not sure if it is convenient to use a folder that's inside the repo for this, without .gitignoring the contents of that folder. 
Other tests just write temporary files to the current working directory; when running the tests via `ctest`, that is the directory that contains the test executable itself, which is somewhere inside the build folder. That's probably more convenient.

Reply:
Yes, you are right. I changed it